sale_product_set/models/product_set.py

```diff
@@ -8,4 +8,13 @@ class ProductSet(models.Model):
 
     name = fields.Char(_('Name'), help=_('Product set name'), required=True)
     set_line_ids = fields.One2many(
-        'product.set.line', 'product_set_id', _(u"Products"))
+        'product.set.line', 'product_set_id', _(u"Products"), copy=True)
```

Comment:
Yeah, it can be another think to check, but for now all is a WIP, and no list is yet built. Pylint config files are auto-documented on the other side.